lasio/reader.py

```diff
@@ -248,23 +248,29 @@ def get_encoding(auto, raw):
     """
     if auto is True:
         try:
-            import cchardet as chardet
-        except ImportError:
+            import pyuchardet as chardet
+        except:
```

Comment:
Can `except ImportError` be retained instead of the bare `except`?

Reply:
Agreed that using except is not ideal. The changes come from the current patch without any modification.  There may have been a reason for using except like this in the patch - I will investigate.

Reply:
@russell-stewart-seequent I have looked at this and think that it is better to leave the code unchanged so that we don't introduce any bugs. We have already made a number of changes as part of this case and at the moment all tests on TeamCity are passing and there don't appear to be any problems.